benchmarks/src/bin/tpch.rs

```diff
@@ -98,6 +98,10 @@ struct DataFusionBenchmarkOpt {
     /// Path to output directory where JSON summary file should be written to
     #[structopt(parse(from_os_str), short = "o", long = "output")]
     output_path: Option<PathBuf>,
+
+    /// Whether to disable collection of statistics (and cost based optimizations) or not.
+    #[structopt(short = "S", long = "disable-statistics")]
```

Comment:
It might be good to keep defaults the same as datafusion-cli (statistics disabled)?

Reply:
I was also thinking about it but then decided against it since the default is already `true`. Do you know whether we have any places where we continuously run benchmarks where making the default `false` would change results? If it wouldn't cause a regression on places where we compare benchmarks across datafusion commits/versions, I think it should be an easy to change.

Reply:
There is no current "continuous benchmarking" in place for this. If that would be the case, it would make sense to run the benchmarks both with and without collecting statistics.

Anyway, good that we can enable / disable them 👍